docs/remote/codespaces.md

```diff
@@ -29,22 +29,22 @@ Example customizations include:
 * Setting environment variables.
 * Configuring editor settings and installing preferred extensions.
 
-See the [Configuring Codespace Environments](https://docs.microsoft.com/visualstudio/codespaces/reference/configuring) documentation for Codespace-specific `devcontainer.json` settings.
+See the [Configuring Codespaces](https://docs.github.com/en/github/developing-online-with-codespaces/configuring-codespaces-for-your-project) documentation for Codespace-specific `devcontainer.json` settings.
```

Comment:
We can remove the `/en/` so that user can get localized content.

Reply:
Done!

Reply:
There are a few more /en/ instances. Can you do a quick search and update the URLs? Thanks

Reply:
I can do a quick scrub with my /api commit